src/app/input-text/page.tsx

```diff
@@ -1,7 +1,13 @@
 'use client';
 
-import TextInput from '@/components/TextInput/TextInput';
+import React from 'react';
+import { InputTitleText, InputText } from '../../components/TextInput/styles'; 
 
-export default function InputText() {
-  return <TextInput />;
-}
+export default function InputTextTest() {
+  return (
+    <>
+      <InputTitleText>Label</InputTitleText>
+      <InputText $error={false} placeholder="Input Suggestion"/>
```

Comment:
Thanks for making this change Catherine! I meant to loop David into this before but forgot— @davidqing6432 can you tell us if you anticipated the label and text input being a single component, like Catherine had before? In the Figma it seemed like this was what was expected, so I just wanted to clarify. 

It seems like the text input never occurs without the label, so it might make sense to make them one component for that reason? Although Jinkang brings up a good point that this allows for more flexibility and is simpler.

Reply:
implemented the combined component as we discussed :)